frontend/Dockerfile

```
@@ -0,0 +1,31 @@
+# Use this image as the platform to build the web
+FROM node:20-alpine AS dockerbuild
+
+# A small line inside the image to show who made it
+LABEL Developers="Curlybytes,DevLeoko"
+
+# https://docs.licensegate.io/self-hosting/#frontend
+# The WORKDIR instruction sets the working directory for everything that will happen next
+WORKDIR /app
+COPY package.json ./
+COPY package-lock.json ./
```

Found a workaround for the requirement of using one entire .env for the docker-based deployment:

```suggestion
COPY ./frontend/package.json ./
COPY ./frontend/package-lock.json ./
```